src/components/tutorialSelector/index.tsx

```diff
@@ -95,6 +95,13 @@ const TutorialSelector: React.FC = (): JSX.Element => {
     }
   `);
 
+  const tutName: string =
+    localStorage.getItem("tutName") ||
+    data.allExampleGqlJson.nodes[0].tutorial_name;
+
+  const tutStepString: string = localStorage.getItem(tutName) || "1";
+  const tutStep: number = parseInt(tutStepString, 10);
+
```

Comment:
I see a bit of redundancy here since you already have this code in the TutorialComponent. Could you try experimenting with turning this into a cutom React Hook and see if it works?

[Here's a little guide.](https://usehooks.com/useLocalStorage/) If you need more help just let me know!